### PR TITLE
ADD mongoCreateSubscription() stub

### PR DESCRIPTION
**Pull request by fgalan**
@crbrox 
@kzangeli

### CODE REVIEW

**Comment by crbrox** [comment]
LGTM

File: src/lib/mongoBackend/mongoCreateSubscription.cpp [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+*
+* Returns:
+* - true: subscription susscessfully created ('oe' must be ignored)
+* - false: subscription creation fail (look to 'oe')
+*/
+bool mongoCreateSubscription
+(
+  const ngsiv2::Subscription&          sub,
+  OrionError*                          oe,
+  std::map<std::string, std::string>&  uriParam,
```
Comment (kzangeli) [inline src/lib/mongoBackend/mongoCreateSubscription.cpp]: **uriParam**  seems like **one** parameter, but this is a map ...

**uriParamV** ?

Comment (fgalan) [inline src/lib/mongoBackend/mongoCreateSubscription.cpp]: Fixed in 51f0af

**Comment by crbrox** [comment]
LGTM

**Comment by kzangeli** [comment]
LGTM

**Pull request merged**